### PR TITLE
Veterinary Crud with exception and jUnit

### PR DESCRIPTION
**Pull request by 1699kratos**
## ISSUE:
Vet CRUD

## FIX:
*Create method for 3 entities
*Read method for 3 entities
*Update method for 3 entities
*Delete method for 3 entities

## Evidence: 
invalidTreatmentDATA
invalidOwnerData
animalInfoSelect
treatmentInfoSelect
ownerInfoSelect

### CODE REVIEW

File: JavaErickHernandez/Veterinary/src/test/java/com/vetpatitas/veterinary/test/OwnerServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+        OwnerDTO ownerDTORequest = new OwnerDTO();
+        ownerDTORequest.setId(1L);
+        ownerDTORequest.setName("GENERIC NAME");
+        ownerDTORequest.setEmail("GENERIC EMAIL");
+        ownerDTORequest.setPhoneNumber("GENERIC PHONE Number");
+
+
+        when(ownerService.deleteOwner(ownerDTORequest));
+        OwnerDTO response = ownerService.deleteOwner(ownerDTORequest);
```
Comment (luisalanmz) [inline JavaErickHernandez/Veterinary/src/test/java/com/vetpatitas/veterinary/test/OwnerServiceImplTest.java:157]: Please verify your delete as I mentioned on Slack

File: JavaErickHernandez/Veterinary/src/test/java/com/vetpatitas/veterinary/test/AnimalTreatmentServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        animalTreatmentDTORequest.setId(1L);
+        animalTreatmentDTORequest.setPetId(1L);
+        animalTreatmentDTORequest.setMedicine("GENERIC NAME");
+        animalTreatmentDTORequest.setDateIn("GENERIC EMAIL");
+        animalTreatmentDTORequest.setDateOut("GENERIC PHONE Number");
+        animalTreatmentDTORequest.setOwner(owner);
+
+
+        when(animalTreatmentService.deleteTreatment(animalTreatmentDTORequest));
+        AnimalTreatmentDTO response = animalTreatmentService.deleteTreatment(animalTreatmentDTORequest);
```
Comment (luisalanmz) [inline JavaErickHernandez/Veterinary/src/test/java/com/vetpatitas/veterinary/test/AnimalTreatmentServiceImplTest.java:178]: please verify in your test the deleted

File: JavaErickHernandez/Veterinary/src/main/java/com/vetpatitas/veterinary/VeterinaryApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.vetpatitas.veterinary;
```
Comment (luisalanmz) [inline JavaErickHernandez/Veterinary/src/main/java/com/vetpatitas/veterinary/VeterinaryApplication.java:1]: 😎  package name

**Review by luisalanmz — COMMENTED** [review]
just one suggestion about your test when you delete something

File: JavaErickHernandez/Veterinary/src/test/java/com/vetpatitas/veterinary/test/OwnerServiceImplTest.java (hunk 1 above)
Comment (1699kratos) [reply JavaErickHernandez/Veterinary/src/test/java/com/vetpatitas/veterinary/test/OwnerServiceImplTest.java:158]: Thanks, already done

**Review by luisalanmz — APPROVED** [review]
LGTM!

**Pull request merged**